website/docs/docs/dbt-cloud-apis/admin-cloud-api.md

```diff
@@ -10,4 +10,4 @@ The dbt Cloud Administrative API is enabled by default for _Team_ and _Enterpris
 - Manage your dbt Cloud account
 - and more
 
-Reference documentation for the dbt Cloud Administrative v2 API can be found [here](/dbt-cloud/api-v2).
+Reference documentation for the dbt Cloud Administrative v2 API can be found [here](/dbt-cloud-apis/api-v2).
```

Comment:
```suggestion
Reference documentation for the dbt Cloud Administrative v2 API can be found [here](/dbt-cloud/api-v2).
```

Reply:
the link points to autogenerated docs. we haven't updated the config for this autogeneration so we need to continue using the old path